README.es-ES.md

```
@@ -0,0 +1,84 @@
+- t4g.[tamaño de instancia]
+- 80GB de almacenamiento
+- imagen de nixOS AARCH64, consulta las amis en [Ami-list](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/virtualisation/amazon-ec2-amis.nix)
+- acceso SSH, con una llave a la que tengas acceso (con el puerto 22 abierto en tu grupo de seguridad)
+
+> Esta es la configuración que utilicé: [configuration.nix](./configuration.nix)
+
+2. accede por ssh a tu servidor
+3. sigue los pasos de [instalación](#installation)
+
+> nota: Debes usar una instancia aarch64, yo usé t4g.2xlarge pero puedes usar una instancia más pequeña si lo deseas.
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)